src/Markdown.MAML/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleToAttribute("Markdown.MAML.Test")]
```

Comment:
nit: new line at the end of the file

Reply:
@vors I don't see a new line at the end of the file, and can't see it in the commit, so I'm going to assume you mean line 2.

Reply:
nope, I mean the end of the file, see https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file 
Git and GitHub are not super happy when dealing with files that doesn't have this extra new line (same goes for C/C++ compilers)

Reply:
Github uses this icon  with a red arrow and crossed circle to mark this problems

Reply:
@vors Thanks for explaining. Didn't know this.

Reply:
Done